scipy/stats/sampling.py

```diff
@@ -39,7 +39,28 @@
    :toctree: generated/
 
    UNURANError
+
+
+Generators for pre-defined distributions
+========================================
+
+To easily apply the above methods for some of the continuous distributions
+in :mod:`scipy.stats`, the following functionality can be used:
+
+.. autosummary::
+   :toctree: generated/
+
+   FastGeneratorInversion
+
 """
+import math
+import warnings
```

Comment:
I have to move the code to a separate file `_sampling.py` to fix the test failures. I think I tried this but `from ._sampling import *` did not work in `sampling.py`.

Reply:
fixed by updating meson.build (thanks @tupui)